ingest/src/main/java/com/connexta/ingest/endpoint/IngestController.java

```diff
@@ -44,7 +46,7 @@ public IngestController(IngestService ingestService) {
     try {
       ingestService.ingest(acceptVersion, fileSize, mimeType, file, title, fileName);
       return new ResponseEntity(HttpStatus.ACCEPTED);
-    } catch (IOException e) {
+    } catch (IOException | HttpClientErrorException | UnknownHttpStatusCodeException e) {
```

Comment:
✏️ It's not clear in this class that these 2 exceptions should be caught. Can the exceptions be caught closer to where we try to hit the transform endpoint or the exceptions included in the method signatures?

Reply:
I agree, it's a little weird how the error handler works since the exceptions aren't checked. 
We could catch it in the client but then we'd still have to rethrow it up so we can return the right exception to the caller.

Reply:
I suggest adding `throws HttpClientErrorException, UnknownHttpStatusCodeException` to `IngestService#ingest` or creating a custom exception like https://github.com/connexta/multi-int-store/pull/64.